code/modules/mob/new_player/login.dm

```diff
@@ -28,9 +28,8 @@
 
 	if(ckey in GLOB.deadmins)
 		verbs += /client/proc/readmin
-	spawn(40)
-		if(client)
-			client.playtitlemusic()
+
+	client?.playtitlemusic()
```

Comment:
Presumably it has a spawn to account for client lag, but thats incredibly dumb. People can have a 2 second lagspike at game start. Its not that bad.

Reply:
If the music is called to play during a lagspike, won't it just start after it anyway? I don't get why the sleep is here at all.